Sources/Tabman/Utilities/Extensions/UIApplication+ExtensionSafe.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  UIApplication+ExtensionSafe.swift
+//  Tabman-Example
+//
+//  Created by Oskar Groth on 2017-12-15.
+//  Copyright © 2017 UI At Six. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+extension UIApplication {
+    
+    static var safeShared: UIApplication? {
+        guard !Bundle.main.bundlePath.hasSuffix("appex") else { return nil }
+        guard UIApplication.respondsToSelector("sharedApplication") else { return nil }
```

Comment:
This needs to be updated for Swift 4, as it currently does not compile under Xcode 9.x. 

E.g. `respondsToSelector()` is `responds(to:)`